### ISSUE TITLE
[BUG]: ZPA : Service Edges - 404 - No handler found 

### ISSUE THREAD

**Issue by LetMeR00t** [mitchos/pyZscaler#87]
Describe the bug
----------------------
Same issue as https://github.com/mitchos/pyZscaler/issues/85 but for Service Edges now

Expected behavior
------------------------
We would expect to have the list of service edges

Screenshots
----------------

    OS: Windows 10
    Browser [e.g. chrome, safari]: Not a browser, it's using the requests library of Python
    Version [e.g. 22]: 2.27.1

Additional context
-------------------------
-

**Comment by mitchos** [comment]
Hi @LetMeR00t I've tried to replicate the issue but I'm unable to get any errors.

My Steps:

1. Link a new service edge and create a service edge group.
2. Run code
3. Both service edge and service edge group were output

My code:

``` python
from pyzscaler import ZPA, ZIA
from pprint import pprint

zpa = ZPA()

for edge in zpa.service_edges.list_service_edges():
    pprint(edge.to_json())

for edge in zpa.service_edges.list_service_edge_groups():
    pprint(edge.to_json())
```

Did you receive an exception or just no output at all?

**Comment by LetMeR00t** [comment]
Hi @mitchos,
Sorry for the late answer,

I have this error on my side:
```
Traceback (most recent call last):
...
  File "......", line 212, in collect_events
    all_data = zpa.service_edges.list_service_edges()
....
restfly.errors.NotFoundError: [404: GET] https://config.private.zscaler.com/userconfig/v1/customers/xxxxxxxx/serviceEdge?page=1 body=b'{\n  "exception" : "No handler found",\n  "id" : "resource.not.found"\n}'

ERROR [404: GET] https://config.private.zscaler.com/userconfig/v1/customers/xxxxxxxx/serviceEdge?page=1 body=b'{\n  "exception" : "No handler found",\n  "id" : "resource.not.found"\n}'
```

I'm not using the v1.0.0 but the latest code from your main branch.

**Comment by mitchos** [comment]
Aha got it. 

I hit this bug earlier when I implemented the SCIM Attributes module. Looks like I didn't bring the change over to the SCIM Groups module. 

Basically what's happening is I overrode the base URL for the ZPA API calls in the ZPA object, rather than in the module itself. This breaks all subsequent API calls after `zpa.scim_groups.list_groups()` or `zpa.scim_groups.get_group()` are invoked. I would bet my money that you are making one of these calls earlier on in your script.

Fix will be out in a few hours in 1.0.2. Thanks for your patience.

**Comment by LetMeR00t** [comment]
Hi,

and you’re right, here is the piece of code using it : https://github.com/LetMeR00t/TA-zscaler-api/blob/main/TA-zscaler-api/bin/input_module_zscaler_zpa_configurations.py
Of course it depends which parameters you select but indeed I’m using this just before 

Thank you
